### ISSUE TITLE
Builds invalid URL when branch name has /

### ISSUE THREAD

**Issue by aelesbao** [ben-gibson/jetbrains-open-in-git-host#18]
When my branch is for example `feature/foo`, the built URL becomes

```
https://github.com/aelesbao/repo/blob/feature%252Ffoo/README.txt
```

**Comment by ben-gibson** [comment]
This should be fixed in version 1.5.4 that has just been released. Could you download the update from your IDE and let me know?

**Comment by aelesbao** [comment]
Tested and it's working fine. Thanks!
